src/Uno.UWP/UI/Colors.cs

```diff
@@ -27,23 +27,21 @@ static partial class Colors
 		/// <returns></returns>
 		public static Color Parse(string colorCode)
 		{
-			if (colorCode?.StartsWith("#", StringComparison.Ordinal) ?? false)
+			if (!string.IsNullOrWhiteSpace(colorCode))
```

Comment:
Passing a whitespace here is very uncommon and will eventually fail too.

```suggestion
			if (!string.IsNullOrEmpty(colorCode))
```

Reply:
Right now whitespace(s) would be treated the same as null or empty.

What input did you have in mind ?

Reply:
Changing `IsNullOrWhiteSpace` to `IsNullOrEmpty` will only change the exception message produced for a whitespace input. I think we don't care right?

The common case for this call is the success case, so we want that as fast as possible, and `IsNullOrEmpty` should be faster than `IsNullOrWhiteSpace`, and we will still throw an exception.

Reply:
I moved the existing

> if (!string.IsNullOrWhiteSpace(colorCode))

code thinking this matched the Windows behaviour :-)

I'll double-check the behaviour on Windows and make sure Uno behave the same.

Reply:
The method is not exposed in UWP nor WinUI so I don't think the behaviour with whitespace would matter.